### ISSUE TITLE
installation full fail

### ISSUE THREAD

**Issue by ppernelle** [claroline/Claroline#684]
I use the composer.full and my parameter.yml to install Claroline 5 
```
git clone https://github.com/claroline/Claroline.git
mv Claroline Claroline5
cd Claroline5
composer require composer/composer dev-master
composer require claroline/bundle-recorder "~5.0" 
cp composer.json.full composer.json
cp ../myparameters.yml app/config/parameters.yml
composer update  --prefer-source
chmod -R 0777 app/cache
chmod -R 0777 app/sessions
chmod 0777 app/config
chmod 0777 app/config/bundles.ini
chmod 0777 app/config/parameters.yml
chmod 0777 app/config/platform_options.yml
chmod 0777 app/config/ips
chmod -R 0777 app/logs
chmod -R 0777 files
chmod -R 0777 web/uploads
chmod -R 0777 web/uploads/logos
``` 
but at the stage

```
php app/console claroline:update
``` 
I have this error
```
[info] Checking database connection...
[info] Creating public sub-directories...
[info] Installing IcapNotificationBundle...
[info] Launching pre-installation actions...
[info] Executing migrations...
[info] Launching post-installation actions...
                                                                                                                                                               
[Doctrine\DBAL\Exception\TableNotFoundException]                                                                                                                 
  An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.version AS version_3, t0.type AS type_4, t0.authors AS authors_5, t0.descrip  
  tion AS description_6, t0.targetDir AS targetDir_7, t0.basePath AS basePath_8, t0.license AS license_9, t0.isInstalled AS isInstalled_10 FROM claro_bundle t0 W  
  HERE t0.name = ? LIMIT 1' with params ["NotificationBundle"]:                                                                                                    
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'claroline5_prod_moocplm.claro_bundle' doesn't exist                                                   
                                                                                                                                                                                                                                                                                     
  [Doctrine\DBAL\Driver\PDOException]                                                                             
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'claroline5_prod_moocplm.claro_bundle' doesn't exist  
                                                                                                                  
  [PDOException]                                                                                                  
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'claroline5_prod_moocplm.claro_bundle' doesn't exist  
                                                                                                                  

claroline:update
```

**Comment by ppernelle** [comment]
ok, config problem in my server

**Comment by ngodfraind** [comment]
php app/console claroline:install should work aswell.
What was the config problem ?

2015-06-01 17:07 GMT+02:00 pernelle <notifications@github.com>:

> Closed #684 <https://github.com/claroline/Claroline/issues/684>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/claroline/Claroline/issues/684#event-318979442>.
>



-- 
Godfraind Nicolas
5, rue de la Carrière
6592 Monceau-Imbrechies
GSM: 0472 94 80 99
